binding.gyp

```diff
@@ -298,6 +298,34 @@
                     'cflags_cc': ['-fexceptions']
                 }]
             ]
+        },
+        {
+          "target_name": "caCentrifugal",
```

Comment:
This looks like you are going to make a different binding target for each compressed air algorithm. It'd be prefered and probably easier if we held all the compressed algorithm bindings under a single target (something like "compressedAir").

Reply:
was not sure on how different the compressors are, 
started with different binding for each,
modified to keep single binding for all compressors